src/libs/actions/Report.ts

```diff
@@ -4845,6 +4846,11 @@ function deleteAppReport(reportID: string | undefined) {
                     [createdAction.reportActionID]: createdAction,
                 },
             },
+            {
+                onyxMethod: Onyx.METHOD.SET,
+                key: ONYXKEYS.SELF_DM_REPORT_ID,
+                value: selfDMReport.reportID,
+            },
```

Comment:
Do we need to set this here? Why?

Reply:
It's not strictly needed. I'll revert this and allow to handle this new onyx key to be handled in separate PR